configs/projects/agent-runtime-5.5.x.rb

```diff
@@ -39,6 +39,6 @@
   proj.component 'rubygem-trollop'
   proj.component 'rubygem-hiera-eyaml'
   proj.component 'ruby-stomp'
-  proj.component 'yaml-cpp' if platform.name =~ /osx-10.12|fedora-29/
+  proj.component 'yaml-cpp' if platform.name =~ /osx-10.12|osx-10.14|fedora-29/
```

Comment:
Interesting, this does not include OSX 10.13. @caseywilliams or @McdonaldSeanp why is that?